src/navigator/enums/screens.enum.ts

```diff
@@ -57,7 +59,8 @@ export type ScreensParamList = {
   [ScreensEnum.ImportAccount]: undefined;
   [ScreensEnum.SyncInstructions]: undefined;
   [ScreensEnum.ConfirmSync]: { payload: string };
-  [ScreensEnum.CreateAccount]: undefined;
+  [ScreensEnum.CreateAccount]: { backupToCloud?: boolean; password?: string; mnemonic?: string };
```

Comment:
it's highly recommended not to send `password` & `mnemonic` through navigation.   
The navigation state is available through context all over the project and some internal lib could easily access those data.